src/Parsers/fuzzers/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 clickhouse_add_executable(lexer_fuzzer lexer_fuzzer.cpp ${SRCS})
-target_link_libraries(lexer_fuzzer PRIVATE clickhouse_parsers)
+target_link_libraries(lexer_fuzzer PRIVATE clickhouse_parsers dbms)
```

Comment:
It's odd that the lexer now depends on dbms. What changed? Maybe it's not needed or maybe we only need common_io? Or is it something else?